tsconfig.dev.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "jsx": "react-jsxdev", // Support JSX in .tsx files, with debug info (e.g. file paths)
```

Comment:
`// ` is technically not valid `.json`. Hopefully the tools that'll ever read this file don't care.